tests/zfs-tests/tests/functional/channel_program/synctask_core/tst.set_props.ksh

```diff
@@ -0,0 +1,40 @@
+#!/bin/ksh -p
```

Comment:
One more thing, it appears neither of these tests were run by the CI since the .ksh and .zcp files were not included in `tests/zfs-tests/tests/functional/channel_program/synctask_core/Makefile.am`.  Please add them.

Reply:
Doh! I thought I found all the places that needed to be updated. It should be good now.